Generator.Application/Models/RequestResult.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Collections.Generic;
+
+namespace Generator.Application.Models
+{
+    public class RequestResult
+    {
+        public RequestResult(List<string> message = null, bool success = true)
+        {
+            Success = success;
```

Comment:
kod w konstruktorach sie troche powtarza, spróbuj je napisać, by jeden korzystał z drugiego, wykorzystując `: this(args)`